deployments/arbitrum/usdc/roots.json

```diff
@@ -3,5 +3,7 @@
     "configurator": "0xb21b06D71c75973babdE35b49fFDAc3F82Ad3775",
     "rewards": "0x88730d254A2f7e6AC8388c3198aFd694bA9f7fae",
     "bridgeReceiver": "0x42480C37B249e33aABaf4c22B20235656bd38068",
-    "bulker": "0xbdE8F31D2DdDA895264e27DD990faB3DC87b372d"
+    "bulker": "0xbdE8F31D2DdDA895264e27DD990faB3DC87b372d",
+    "arbitrumCCTPTokenMinter": "0xE7Ed1fa7f45D05C508232aa32649D89b73b8bA48",
+    "arbitrumCCTPMessageTransmitter": "0xC30362313FBBA5cf9163F0bb16a0e01f01A896ca"
```

Comment:
nit: don't think we need to prepend with `arbitrum` to the names here since the this is the roots for arbitrum.

Reply:
Also I don't think these need to be roots on the Arbitrum deploy.

Reply:
Removed 👍